python_ta/cfg/visitor.py

```diff
@@ -126,8 +126,9 @@ def visit_functiondef(self, func: nodes.FunctionDef) -> None:
             )
         )
 
-        self._current_cfg.start.add_statement(func.args)
-        func.cfg_block = self._current_cfg.start
+        # Current CFG block is self._current_cfg.start while initially creating the function cfg
+        func.args.accept(self)
```

Comment:
This isn't necessary, and I suspect is now causing a problem with the tests. The `Arguments` node is being added in the `add_arguments` method

Reply:
Another note is that under `patches/transforms.py`, you can see that any exceptions when using `CFGVisitor` are being caught but unreported. You could try removing the try-except and just keeping the `ast.accept` call and seeing if you see more useful errors when you run the tests